pkg/lang/csharp/runtimes/aws/Lambda_Dispatcher.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+using Amazon.Lambda.Core;
+
+namespace KlothoRuntime
+{
+    public class APIGatewayLambdaDispatcher : {{if .Expose.APIGatewayProxyFunction}}{{.Expose.APIGatewayProxyFunction}}{{else}}{{.Expose.FunctionType}}{{end}}
```

Comment:
This doesn't seem like valid C#. The idea with the `//TMPL` and compile_template was so that these files could still be parsed and operated on as normal C# files (to make them easier to maintain). If there's no intention to keep this as valid C#, then we should just use the `.cs.tmpl` file directly.

Reply:
`//TMPL` for template control flow logic just makes it a lot easier to read.
<img width="608" alt="image" src="https://user-images.githubusercontent.com/110404901/220744235-ec5d3162-12fe-44f9-855b-8aac90af7549.png">
<img width="607" alt="image" src="https://user-images.githubusercontent.com/110404901/220744324-ded939fb-60ad-47b4-b801-9b95c66696f0.png">

Reply:
Is it the indentation or the colouring that makes it easier to read?

Reply:
The coloring.

That said, I get that eliminating the `.cs` file eliminates a bunch of overhead when making changes.